taca/analysis/analysis.py

```diff
@@ -159,6 +160,12 @@ def transfer_runfolder(run_dir, pid, exclude_lane):
     :param: string exclude_lane: lanes to exclude separated by comma
 
     """
+    # Validate whether run_dir exists or is valid
+    run_dir = os.path.abspath(run_dir)
```

Comment:
Do you on purpose overwrite `run_dir`?

Reply:
Yes, this is to be sure that `run_dir` will always be the full path